src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
+  span.iui-right {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
+    }
+  }
+  
+  span.iui-left {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  span.iui-right {
+    margin-left: $iui-s;
+  }
+}
+
+@mixin iui-slider-container {
+  display: flex;
+  position: relative;
+  z-index: 1; // above ticks
+  width: 100%;
+  pointer-events: none;
+  flex-direction: column;
+  justify-content: center;
+
+  .iui-tooltip {
+    display: none;
+  }
+
+  // hidden input[type=range] that is used to process events and hold value
+  .iui-slider {
```

Comment:
so if it is supposed to be hidden, cant we just apply same rules like in other custom input elements (checkbox, radio, etc...)?
Now there is a bunch of those and seems only appearance does the difference..

Reply:
oh, i see for it to work need to have input same as the visual one (meaning 100% width, so it covers whole slider and can react to events..)